packages/helper-grammar-regex-collection/index.js

```diff
@@ -201,7 +201,7 @@ export const JAVA_IMPORT = regex`
 `;
 
 export const NET_PROJ_PACKAGE = regex`
-  <(PackageReference|DotNetCliToolReference|FrameworkReference)
+  <(PackageReference|DotNetCliToolReference|FrameworkReference|Reference)
```

Comment:
This could be switched to something like this but I'm not sure if it's worth it. Thoughts?

```suggestion
  <(DotNetCliTool|Framework|Package)?Reference
```

Reply:
👍 I like it